tests/core/theme/ThemeManagerTest.php

```diff
@@ -13,6 +14,32 @@
  */
 class ThemeManagerTest extends LuyaWebTestCase
 {
+    public function testSetup()
+    {
+        $themeManager = new ThemeManager();
+        $themeManager->activeThemeName = '@app/themes/blank';
+        $themeManager->setup();
+        
+        $this->assertNotNull(Yii::$app->view->theme, 'Theme must be set.');
+        $expectedPath = realpath(Yii::getAlias('@luyatests/data/themes/blank'));
+        $this->assertEquals($expectedPath, Yii::$app->view->theme->basePath, 'Theme base path not correct.');
+        $this->assertEquals($expectedPath, Yii::getAlias('@activeTheme'), 'Alias path is not correct.');
+    
+        $this->assertInstanceOf(Theme::class, $themeManager->getActiveTheme());
+    }
+    
+    public function testSetupWithoutActiveTheme()
```

Comment:
what if you add 

```php
/**
    * @runInSeparateProcess
    */
 ```

https://phpunit.de/manual/3.7/en/appendixes.annotations.html#appendixes.annotations.runInSeparateProcess

Reply:
Yes of course 🙈 ...